### PR TITLE
docs(troubleshooting.md): run non-headless on CI

### PR DESCRIPTION
**Pull request by mshustov**
related issue: https://github.com/GoogleChrome/puppeteer/issues/3303

### CODE REVIEW

**Comment by aslushnikov** [comment]
@restrry can you please sign a CLA to make our bot happy?

**Comment by mshustov** [comment]
@aslushnikov done!

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -101,6 +101,14 @@ To run headless Chrome on Travis, you *must* call `launch()` with flags to disab
 const browser = await puppeteer.launch({args: ['--no-sandbox']});
 ```
 
+Some Puppeteer functionality (like work with Chrome extensions) may operate only in non-headless mode. To be able to run Puppeteer in non-headless mode on CI you need to run browser virtually. For that, you can use [Xvfb](https://en.wikipedia.org/wiki/Xvfb) server
```
Comment (vsemozhetbyt) [inline docs/troubleshooting.md]: Nit: maybe a colon at the end of the paragraph?

Comment (mshustov) [reply docs/troubleshooting.md]: fixed

**Pull request merged**